kornia/augmentation/random_generator/_2d/color_jiggle.py

```diff
@@ -54,7 +56,7 @@ def __repr__(self) -> str:
         )
         return repr
 
-    def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
+    def make_samplers(self, device: Device = None, dtype: Dtype = None) -> None:
```

Comment:
```suggestion
    def make_sampler(self, device: Device = None, dtype: Dtype = None) -> None:
```
thinking in a more generic way to customize you own sampler